Documentation/upgrade.md

```diff
@@ -14,8 +14,8 @@ The goal is to provide prescriptive guidance and knowledge on how to upgrade a l
 We welcome feedback and opening issues!
 
 ## Supported Versions
-The supported version for this upgrade guide is **from an 0.7 release to the latest builds**. Until 0.8 is released, 
```

Comment:
there is another place with this wording of "Until 0.8 is released..." below in the "Upgrade Process" section.

Reply:
good catch, i think that whole sentence can be removed.